src/main/scala/rocks/muki/graphql/GraphQLQueryPlugin.scala

```diff
@@ -25,9 +25,7 @@ object GraphQLQueryPlugin extends AutoPlugin {
   import autoImport._
   import GraphQLSchemaPlugin.autoImport._
 
-  // TODO separate these into two auto plugins
-  override def projectSettings: Seq[Setting[_]] =
-    pluginSettings(Compile) ++ pluginSettings(IntegrationTest)
+  override def projectSettings: Seq[Setting[_]] = pluginSettings(Compile)
 
   private def pluginSettings(config: Configuration): Seq[Setting[_]] =
     inConfig(config)(
```

Comment:
The config can be removed from the following line below:

         graphqlQueryDirectory := (sourceDirectory in Compile).value / "graphql",

Reply:
Good catch. I'm thinking about putting things into the `Test` scope as these are actually tests.

I'm also less and less sure if the `QueryPlugin` is actually necessary. You can write simple unit
tests with scalatest and sangria pretty easy as well. WDYT?

Reply:
The QueryPlugin helps to check that the generated schema and associated queries. I guess that may be helpful in cases where you don’t us Scala end-to-end. In the project I used sangria in we had a scala.js client and was able to write unit tests that ran queries against the server-side and also tested the JSON serialization.

Reply:
Interesting point. Then let's keep it. I think the test scope is a better fit then, too.